src/api/hooks/useSubmitTransaction.ts

```diff
@@ -40,8 +40,7 @@ const useSubmitTransaction = () => {
 
   async function submitTransaction(transaction: InputTransactionData) {
     if (
-      network?.name.toLocaleLowerCase() !==
-        (state.network_name === "local" ? "localhost" : state.network_name) &&
+      network?.name.toLocaleLowerCase() !== state.network_name &&
```

Comment:
the network name Petra responds with is Localhost though :(

Reply:
From my tests it is `local`, and it also matches to the adapter NetworkInfo type that Explorer is using
```
export type NetworkInfo = {
  name: Network;
  chainId?: string;
  url?: string;
};
```

Reply:
How did you test?

<img width="509" alt="Screenshot 2025-03-06 at 12 00 15 PM" src="https://github.com/user-attachments/assets/6e3d54eb-d02a-496d-b36a-d25d0b91787c" />

Reply:
Through Explorer, it uses the wallet adapter that standardize the network name - `local`.